package.json

```diff
@@ -1993,6 +1993,12 @@
                     "default": false,
                     "description": "Creates separate output panels for kernels/jupyter server console output",
                     "scope": "machine"
+                },
+                "jupyter.excludeUserSitePackages": {
+                    "type": "boolean",
+                    "default": false,
+                    "description": "Add PYTHONNOUSERSITE to kernels before starting. This prevents global/user site-packages from being used in the PYTHONPATH of the kernel.",
```

Comment:
Added this is an option in case we break anybody. We can have them set this to try the old way. Telemetry should tell us how many people have this set.